### DISCUSSION TITLE
캐릭터 라이팅

### DISCUSSION THREAD

**Discussion by xowjd913** [naverz/zepeto-studio-kor#1182] [5. Q&A]
안녕하세요.

현재 월드에서 Directional Light를 쓰는데 
특정 위치에서 캐릭터의 얼굴에 그림자가 너무 심하게 생겨서
캐릭터에 라이팅을 주지않고 싶습니다.

방법이 있을까요?

**Comment by hjs6082** [comment]
디렉셔널 라이트 옵션에서 Shadow를 No Shadow로 바꿔보세요!
